README.md

```diff
@@ -42,6 +42,18 @@ yay tfswitch
 # precompiled
 yay tfswitch-bin
 ```
+
+### Scoop for Windows
+
+Installation for Windows via [`scoop`](https://scoop.sh/), but currently unstable[^win-unstable]
+
+```sh
+scoop bucket add main
+scoop install terraform-switcher
+```
+[^win-unstable]: Complete windows support is dependant on [#267](https://github.com/warrensbox/terraform-switcher/pull/267) and [#262](https://github.com/warrensbox/terraform-switcher/pull/262)
```

Comment:
Could this annotation be moved to where it appears in this PR? I think it would be a way more convenient as opposite to having it as a real foot-note at the ver end of the page. Thanks.

Reply:
I have made the modifications